rfcs/src/rfcs/0001-mir-linker.md

```diff
@@ -0,0 +1,244 @@
+- **Feature Name:** `MIR` Linker (mir_linker)
+- **RFC Tracking Issue**: <https://github.com/model-checking/kani/issues/1588>
+- **RFC PR:** <https://github.com/model-checking/kani/pull/1600>
+- **Status:** Under Review
+- **Version:** 0
+
+-------------------
+
+## Summary
+
+Fix linking issues with the rust standard library in a scalable manner by only generating `goto-program` for
```

Comment:
Another nit: I'd use "goto-program(s)" (see [CBMC docs](http://cprover.diffblue.com/group__goto-programs.html)) instead since `goto-program` is not a tool or module. You frequently say "`goto-program` model" but I think "goto-program" and "goto model" are the same thing.